app/services/flowcommerce_spree/order_updater.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module FlowcommerceSpree
+  class OrderUpdater
+    def initialize(order:)
+      raise(ArgumentError, 'Experience not defined or not active') unless order.zone&.flow_io_active_experience?
+
+      @experience = order.flow_io_experience_key
+      @order = order
+      @client = FlowcommerceSpree.client
+    end
+
+    def upsert_data(flow_io_order = nil)
+      flow_io_order ||= @client.orders.get_by_number(FlowcommerceSpree::ORGANIZATION, @order.number).to_hash
+
+      puts "Flow IO Order #{flow_io_order}"
+
+      @order.flow_data['order'] = flow_io_order
+      attrs_to_update = { meta: @order.meta.to_json }
+      if @order.flow_data.dig('order', 'submitted_at').present? && !@order.complete?
+        # flow_io_total_amount = order.flow_io_total_amount&.to_d
```

Comment:
Please delete all this dead code 🙏

Reply:
ohh yesyes!! I haven't de;ete any of it just in case. but I will make sure to leave this clean and rubocop/flay warnings free. Just pushing code for test in dev5 only. Thanks for pointing this out though!!! :D